Add a step for manually enabling el8 modules during upgrade (#1900)
theforeman/foreman-documentation: guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

# {foreman-maintain} self-upgrade
----

> ifdef::satellite[]

Reviewer: ```suggestion
```

This is already in the Satellite section, no need to add another `ifdef`.

> If the {Project}-{ProductVersionPrevious} system is upgraded from RHEL7 to RHEL8 using using LEAPP, manually enable the Satellite module:

Reviewer: ```suggestion
If the {SmartProxy} {ProductVersionPrevious} system was upgraded from {RHEL} 7 to {RHEL} 8 using Leapp, manually enable the {SmartProxy} module:
```

+
[options="nowrap" subs="attributes"]
----

> # dnf module enable satellite:el8

Reviewer: ```suggestion
# dnf module enable satellite-capsule:el8
```

The module is called `satellite-capsule` for capsules.

----

> endif::[]

Reviewer: ```suggestion
```

This is already in the Satellite section, no need to add another `ifdef`.
